scripts/fill-messages-to-influx.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+var path = require('path'),
+    chalk = require('chalk'),
+    async = require('async'),
+    config = require(path.resolve('./config/config')),
+    configMongoose = require(path.resolve('./config/lib/mongoose')),
+    mongoose = require('mongoose'),
+    fluxIn = require(path.resolve('./modules/messages/server/services/message-to-influx.server.service')),
+    messageModels = require(path.resolve('./modules/messages/server/models/message.server.model')),
+    Message = mongoose.model('Message');
+
+    // the expression below uses ES6 Promises, so it shouldn't belong here.
+    // there is a warning when one doesn't provide one's own library.
+    // it was present elsewhere i believe
+    mongoose.Promise = Promise;
+
+// Bootstrap db connection
+var db = mongoose.connect(config.db.uri, function(err) {
+  if (err) {
+    console.log(chalk.red('Could not connect to MongoDB!'));
+    console.error(err);
+  }
+});
+
+async.waterfall([
+
+  function readAllMessages(done) {
+    Message.find(done);
```

Comment:
Is this meant to only be used in development?

If we wanted to run this on production it might make sense to use mongoose's stream support instead of retrieving all the records in the entire messages collection and then using `.forEach()` to iterate over them.